schemas/json/layout/layout.schema.v1.json

```diff
@@ -182,6 +189,49 @@
         }
       }
     },
+    "gridValue": {
+        "type": "integer",
+        "exclusiveMaximum": 13,
+        "exclusiveMinimum": 0,
+        "examples": [12]
+    },
+    "gridSettings": {
+        "allOf": [{ "$ref": "#/definitions/gridProps"}],
```

Comment:
Yes, `To validate against allOf, the given data must be valid against all of the given subschemas.`. I guess both `anyOf `and `allOf` would be valid here since we have only one sub-schema.